scss/_typography.scss

```diff
@@ -48,6 +48,7 @@ h6,
   margin: $heading-margins;
   font-family: $font-family-sans-serif;
   color: $dark-silver;
+  color: inherit;
```

Comment:
Nice 👍🏻This will make sure headers are accessible by default when there's a background color.

Reply:
Are both color properties necessary? The lower one will just override the color: $dark-silver one.